x-pack/plugins/apm/server/routes/errors/get_error_groups/get_error_group_main_statistics.ts

```diff
@@ -72,6 +79,19 @@ export async function getErrorGroupMainStatistics({
     ? { [maxTimestampAggKey]: sortDirection }
     : { _count: sortDirection };
 
+  const shouldQuery = searchQuery
+    ? {
+        should: [
+          ...wildcardQuery(ERROR_LOG_MESSAGE, searchQuery),
+          ...wildcardQuery(ERROR_EXC_MESSAGE, searchQuery),
+          ...wildcardQuery(ERROR_GROUP_NAME, searchQuery),
+          ...wildcardQuery(ERROR_EXC_TYPE, searchQuery),
+          ...wildcardQuery(ERROR_CULPRIT, searchQuery),
+        ],
```

Comment:
Maybe this would avoid duplicating the wildcardQuery call. But it's up to you, just a thought.
```suggestion
        should: [
          ERROR_LOG_MESSAGE,
          ERROR_EXC_MESSAGE,
          ERROR_GROUP_NAME,
          ERROR_EXC_TYPE,
          ERROR_CULPRIT,
        ].map((field) => ({
          ...wildcardQuery(field, searchQuery),
        })),
```

Reply:
lgtm 👍

Reply:
I think this is better

```suggestion
        should: [
          ERROR_LOG_MESSAGE,
          ERROR_EXC_MESSAGE,
          ERROR_GROUP_NAME,
          ERROR_EXC_TYPE,
          ERROR_CULPRIT,
        ].flatMap((field) => wildcardQuery(field, searchQuery)),
```